### PR TITLE
Utility endpoint

### PR DESCRIPTION
**Pull request by kevxo**
Description:  Created utility end-point with index of all businesses and show of one business. Using the webmock to mock the api call. This includes updates to the routes and new controllers. Refactored the API calls to Service, Facade, and Poro.
Moving forward is to try to actually hit the api call from backend
* 

Fixes #

* [x] All tests passing
* [x] Lint code
* [x] Checked local host
* [x] Updated README

### CODE REVIEW

File: app/facades/search_facade.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    json[:data].map do |biz_data|
+      Business.new(biz_data)
+    end
+  end
+
+  def self.a_business(location, category, type, id)
+    json = CategoryService.a_business(location, category, type, id)
+    Business.new(json)
+  end
+end
```
Comment (BJSherman80) [inline app/facades/search_facade.rb:13]: Nice job here

**Review by BJSherman80 — APPROVED** [review]

**Comment by jakeheft** [comment]
Kevin, can you check off the boxes in your PR info that you completed (e.g. all tests passing, lint code, etc.)

File: app/views/service_types/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<h1><%= params[:type] %> Businesses</h1>
+
+<% @businesses.each do |biz| %>
+	<div class='business'>
+		<%= link_to biz.name, "/#{params[:location]}/#{params[:service]}/#{params[:type]}/#{biz.id}" %>
```
Comment (Oxalisviolacea) [inline app/views/service_types/index.html.erb:5]: We're going to change these to buttons for more visibility on the page.

**Review by Oxalisviolacea — APPROVED** [review]

File: app/views/service_types/index.html.erb (hunk 2 above)
Comment (Stearnzy) [inline app/views/service_types/index.html.erb:5]: Could maybe replace with <%= link_to biz.name, business_show_path %> or something like that?

**Pull request merged**